python/tvm/relay/op/contrib/dnnl.py

```diff
@@ -85,3 +87,76 @@ def pattern_table():
     conv2d_relu_pat = ("dnnl.conv2d_relu", make_pattern(with_bias=False))
     dnnl_patterns = [conv2d_bias_relu_pat, conv2d_relu_pat]
     return dnnl_patterns
+
+
+def AutoQuery(N,IC,KH,KW,OC,SH,SW,PH_L,PH_R,PW_L,PW_R,OH,OW,G):
```

Comment:
If this function is limited for conv2d, suggest to change the function name to more meaningful one like "get_optimal_layout_for_conv2d()"? Similar to the one in "ffi_api.AutoQuery"?

Reply:
changed name to get_optimal_layout_for_conv2d().